packages/cache-browser-local-storage/src/types/BrowserLocalStorageOptions.ts

```diff
@@ -4,6 +4,11 @@ export type BrowserLocalStorageOptions = {
    */
   readonly key: string;
 
+  /**
+   * The time to live for each cached item in seconds.
+   */
+  readonly ttl?: number;
```

Comment:
I guess we can go full length here and use `timeToLive` to make it easier for everyone, wdyt?

Reply:
Took a look at some of the other `Options.ts` files, and I do agree with this suggestion.
Not only does it make it a bit easier to read, it also makes it more consistent with other configurations.

Reply:
It has been updated 👍